susanssky/careless-whisper
# 52-deleting transcription task fe and be completed!

### Description

This PR adds the capability to delete a  transcription/post in both the backend and frontend of the application. It enables users to remove a specific one through a dedicated delete button.

- **Testing** - 

- `npx prisma migrate dev --name init`
-  login with github
- click on any post you want to delete to move to the related page
- click on the delete button
- you should be directed to the dashboard as the post is deleted. You might need to refresh the page to see the result (disappearing the post) on the table in dashboard.
